stock_picking_package_preparation_line/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+#    Author: Francesco Apruzzese
```

Comment:
@SimoRubi Remove copyright from all `__init__.py` file.

Reply:
Thanks @nikul-serpentcs for your review, but this PR is still WIP (it says that in the title).
I haven't migrated this yet because I'm waiting for https://github.com/OCA/stock-logistics-workflow/pull/448 to be merged, could you please review that instead?

I'll look at your suggestions when I migrate this.

Reply:
Ok. I will review #448 and if it's merged then attended other those also.